src/lib/nodes/adm/index.ts

```diff
@@ -1,8 +1,11 @@
 import config from '@/config'
-import { NodeInfo } from '@/types/wallets'
+import type { NodeInfo } from '@/types/wallets'
 import { AdmClient } from './AdmClient'
 
-const endpoints = (config.adm.nodes.list as NodeInfo[]).map((endpoint) => endpoint.url)
+const endpoints = (config.adm.nodes.list as NodeInfo[]).map((endpoint) => ({
+  alt_ip: endpoint.alt_ip,
+  url: endpoint.url
+}))
```

Comment:
Don't need mapping. The `endpoint` is matching the `NodeInfo` type. Check other places as well.

```suggestion
const endpoints = (config.adm.nodes.list as NodeInfo[]))
```

Reply:
Although mapping derived from dev now this is redundant indeed

Reply:
https://github.com/Adamant-im/adamant-im/pull/768/commits/9a5e3afff024e4687eba55a6cd07ae42eb6bcb58